src/main/java/seedu/address/logic/commands/CropCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.core.Config.ASSETS_FILEPATH;
+import static seedu.address.commons.core.Config.TEMP_FILEPATH;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import javax.imageio.ImageIO;
+
+import org.apache.commons.io.FileUtils;
+import org.imgscalr.Scalr;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.image.Image;
+
+/**
+ * This command allows the user to crop an image.
+ */
+public class CropCommand extends Command {
+
+    public static final String COMMAND_WORD = "crop";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Crop the image according to the coordinates given.\n"
+            + "Parameters: X-COORDINATE, Y-COORDINATE, WIDTH, HEIGHT \n"
+            + "All values to be integer/whole numbers only. \n"
+            + "Example: " + COMMAND_WORD + " 0 0 200 200 sample.png";
+    private int xCoord;
+    private int yCoord;
+    private int width;
+    private int height;
+    private String fileName;
+
+    public CropCommand(int x, int y, int width, int height, String image) {
+        xCoord = x;
```

Comment:
A suggestion to store your coordinates is to store them as tuples, where you can easily if 2 coordinates are equal and access the x and y values as `.x` or `.y`.